Osmalyzer/Analyzers/Parcel Lockers/SmartpostParcelLockerAnalyzer.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+namespace Osmalyzer;
+
+public class SmartpostParcelLockerAnalyzer : ParcelLockerAnalyzer<SmartpostParcelLockerAnalysisData>
```

Comment:
I renamed it to `Itella` since technically Itella LV is the operator and "Smartpost" is their parcel network brand. Mainly did this because I externalized branding data so I can reuse it in a different analyzer, but I wanted to ensure any tagging I potentially check in the future is accurate from there.

Reply:
I used brand instead of the operator in the name of the class, even though it was technically incorrect for some fields, because it made more sense for future Unisend/uDrop combo. 
I'm not sure how to proceed with them now.

Reply:
I'm not sure in this. 
RN I'm under impression that Unisend is both a brand and and operator. And uDrop is a separate operator, working under same brand. 
At least that's how I explain to myself why there are two different sites, and one shows all lockers, and another - only those of uDrop.
And additionally, branding of lockers differs: Just unisend ones are yellow and have only unisend written on them, and the other ones are white with branding for both unisend and udrop.

Reply:
Oh, geez, you're right, I feel dumb now. I didn't realize it's not the same company doing it because they are all currently used by Smartpost. What a mess. Let me just summarize for posterity and to see if I understand everything.

So there are parcel lockers by operator/company Unisend. Local company Unisend Latvia (owned by Lithuanian Lietuvos paštas). Yellow/some green. These don't have additional branding (or it's just "Unisend").

Then there are parcel lockers by operator/company uDrop. Local company Udrop LV. Green/white. These do not themselves operate a parcel network, but other delivery network operators can use these as drop-off/pick-up points. In Latvia, AFAIK, currently, only Unisend is using them and all parcel lockers are also prominently (confusingly) advertised as Unisend.

So technically, I guess, we should only be parsing the delivery network operator delivery points (parcel lockers) - Unisend, in this case. If they happen to use uDrop, then we can add/count and verify uDrop-specific lockers for them too. Luckily, Unisend lists and differentiates their uDrop lockers on their site.

At the same time, we can parse and check uDrop lockers, regardless of what delivery networks are using them - basically see if any are missing or mistagged. But not assume they are part of any specific "brand". I guess this will avoid a terrible mess in case more companies start using uDrop.

Reply:
Yes, this is more or less my understanding too. I'm not sure about part where other companies/networks can or cannot use lockers of uDrop, but at this moment I feel sufficiently exhausted with shenanigans of that network to properly investigate. 
I'll try to add analyzer based on data from unisend's site, and then we'll see if it requires adjustment later.
One thing that needs an alignment: how both of them should be tagged? 
I assumed `brand=Unisend` + `operator=Unisend` and `brand=Unisend` + `operator=uDrop` (and already mapped a couple in that way). But maybe this is a correct usecase for [`network=Unisend`](https://wiki.openstreetmap.org/wiki/Key:network#Amenities)?